typescript/src/feast/pubsub/pubsub.ts

```diff
@@ -1,6 +1,41 @@
-import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { parsePayload, toSqsSubReference } from "../../pubsub/apple";
+import { AppleSubscriptionReference } from "../../models/subscriptionReference";
+import Sqs from 'aws-sdk/clients/sqs';
+import { sendToSqs } from "../../utils/aws";
+import { AWSError } from "aws-sdk";
+import { PromiseResult } from "aws-sdk/lib/request";
+import { HTTPResponses } from "../../models/apiGatewayHttp";
 
 export async function handler(request: APIGatewayProxyEvent): Promise<APIGatewayProxyResult>  {
     console.log(`[34ef7aa3] ${JSON.stringify(request)}`)
-    return { statusCode: 200, body: JSON.stringify(request) }
+    return processEvent()(request)
```

Comment:
Super nitpicky, but I wonder if a slightly better name for the outer function application would be something like `buildProcessEvent`, and we could even separate out the calls to make the intention really clear:

```suggestion
    const processEvent = buildProcessEvent();
    return processEvent(request);
```

What do you reckon? (I don't feel strongly - feel free to ignore!)